android-components/components/feature/tabs/src/main/java/mozilla/components/feature/tabs/TabsUseCases.kt

```diff
@@ -426,15 +439,22 @@ class TabsUseCases(
          * @param private Whether or not this session should use private mode.
          * @param source The origin of a session to describe how and why it was created.
          * @param flags The [LoadUrlFlags] to use when loading the provided URL.
+         * @param ignoreFragment Whether to ignore the fragment/anchor of the url while comparing
```

Comment:
```suggestion
         * @param ignoreFragment Whether to ignore the fragment identifier of the url while comparing
```
Let's remove the ambiguity here since we know the official name for this.